Rover/radio.cpp

```diff
@@ -146,6 +146,9 @@ void Rover::radio_failsafe_check(uint16_t pwm)
 {
     if (!g.fs_throttle_enabled) {
         // radio failsafe disabled
+        if (notify.flags.failsafe_radio) {
+        AP_Notify::flags.failsafe_radio = false;
```

Comment:
thanks for this.  we can remove the "if" part of this and simply just always set the flag.

Reply:
will do